client/prototypes.h

```diff
@@ -543,6 +543,11 @@ TDNFConfGetRpmVerbosity(
     PTDNF pTdnf
     );
 
+uint32_t
+TDNFParseOSInfo(
```

Comment:
You can make this a static function in `config.c`.

Reply:
Can we use config parser for parsing `/etc/os-release`?

Also, we have some info on OS with `uname` APIs? can we use that here as well?

Reply:
> Can we use config parser for parsing /etc/os-release?

No.

> Also, we have some info on OS with uname APIs? can we use that here as well?

I don't think it's needed.